library/ssl_srv.c

```diff
@@ -21,6 +21,8 @@
 
 #if defined(MBEDTLS_SSL_SRV_C)
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_2)
```

Comment:
The "#if defined(MBEDTLS_SSL_PROTO_TLS1_2)" pre-processor directives have not been removed inside the file.

Reply:
For tls13_only , the entire file can be ignore